tests/python/regression_test_utilities.py

```diff
@@ -53,14 +53,19 @@ def get_sparse():
     X = sparse.csr_matrix(X)
     return X, y
 
-
```

Comment:
Need two empty lines between Python functions, according to existing code standard. Otherwise lint test will fail.